libexternal/src/osxsupport.cpp

```diff
@@ -39,7 +39,7 @@ char *string_to_utf8(const char *p_string)
 	if (s_text_to_unicode_info != NULL)
 	{
 		unsigned int t_string_length;
-		t_string_length = strlen(p_string);
+		t_string_length = (unsigned int)strlen(p_string);
```

Comment:
This is an unchecked overflow on 64-bit systems.  It would be better to refactor with `size_t t_string_length;`